pkg/volume/csi/csi_attacher.go

```diff
@@ -318,6 +318,15 @@ func (c *csiAttacher) MountDevice(spec *volume.Spec, devicePath string, deviceMo
 	}
 	publishVolumeInfo := attachment.Status.AttachmentMetadata
 
+	nodeStageSecrets := map[string]string{}
+	if csiSource.NodeStageSecretRef != nil {
+		nodeStageSecrets, err = getCredentialsFromSecret(c.k8s, csiSource.NodeStageSecretRef)
+		if err != nil {
+			return fmt.Errorf("get secret %s/%s failed: %v",
```

Comment:
nit: `"fetching NodeStageSecretRef failed for secret %s/%s failed: %v`

Reply:
Thanks for reminding, already fixed.